Core/GDCore/Extensions/Builtin/AdvancedExtension.cpp

```diff
@@ -34,6 +34,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAdvancedExtension(
           "res/function32.png")
       .SetHelpPath("/events/functions/return")
       .AddParameter("expression", "The number to be returned")
+      .SetReleventForFunctionEventsOnly()
```

Comment:
Typo: Relevent => Relevant

Reply:
Should we use the same wording as "scope"? Like: "SetScopedToFunctionEventsOnly"?

Reply:
I'm not sure. It's a rule that apply on the scope but so do visibility and the property is called `IsPrivate` but not `IsScopedToExtension`.
I think it make less sense to use "scoped" in the context of this PR because the rules is more on the type of scope rather than a specific scope.